CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [FIXED] Allow $and, $or, and $nor operator selectors
```

Comment:
nit: might be nice to backtick the operators for markdown

Reply:
Done in 500a660.